compose/foundation/foundation/src/desktopMain/kotlin/androidx/compose/foundation/ContextMenuProvider.desktop.kt

```diff
@@ -224,11 +224,9 @@ class ContextMenuData(
     val next: ContextMenuData?
 ) {
 
-    internal val allItems: List<ContextMenuItem> by lazy {
-        allItemsSeq.toList()
-    }
+    internal val allItems: List<ContextMenuItem> get() = allItemsSeq.toList()
 
-    internal val allItemsSeq: Sequence<ContextMenuItem>
+    private val allItemsSeq: Sequence<ContextMenuItem>
```

Comment:
Not really related to the change in this PR, but why do we need a sequence here? We could do the same with
```
    internal val allItems: List<ContextMenuItem>
        get() = buildList {
            var current: ContextMenuData? = this@ContextMenuData
            while (current != null) {
                addAll(current.items())
                current = current.next
            }
        }
```

Reply:
Your code is better